README.md

```diff
@@ -486,7 +488,8 @@ oct2 = 0o755 # useful for Unix file permissions
 bin1 = 0b11010110
 ```
 
-64 bit (signed long) range expected (−9,223,372,036,854,775,808 to
+TOML implementations must be able to losslessly handle at least all integers in
+the 64 bit (signed long) range (−9,223,372,036,854,775,808 to
 9,223,372,036,854,775,807).
```

Comment:
Would you be willing to drop this change, for now? We'll tackle this in a follow up, and I'd prefer to not hold up the entire PR over a single point of disagreement. :)

Reply:
Sure, I've dropped that change for now.

Reply:
Yay! Thank you! ^>^